cmd/config/internal/commands/cmdset_test.go

```diff
@@ -403,10 +449,9 @@ spec:
 `,
 			errMsg: "tag in body should be at least 6 chars long",
 		},
-
 		{
 			name: "validate openAPI list values",
-			args: []string{"list", "10", "hi", "true"},
+			args: []string{"--name", "list", "--list-value", "10", "--list-value", "hello", "--list-value", "true"},
```

Comment:
This might be tedious for users to mention flag for each value. It would be better to stick to current way of pulling list values.

Reply:
reverted.